### ISSUE TITLE
Buffer overflow exception when input contains 'z's

### ISSUE THREAD

**Issue by zaxdo** [fzakaria/ascii85#4]
When decoding, the formula for the output size is "input_size * 4 / 5", which does account for the fact that when input contains only 'z's output can be 4 times as large as the output.

**Comment by fzakaria** [comment]
@zaxdo not sure i understand.

I tried this:

```
    @Test
    public void testOnlyZInput() {
        String payload = "zzzz";
        String encoded = Ascii85.encode(payload.getBytes());
        assertThat(payload, is(new String(Ascii85.decode(encoded), StandardCharsets.US_ASCII)));
    }
```

**Comment by zaxdo** [comment]
Do Ascii85.decode("zzzzzzzzzzzzzzzzzzzzzz");

**Comment by fzakaria** [comment]
Fixed this using #10 with the help of @odisseus 
Closing this issue.

**Comment by fzakaria** [comment]
I also released a new version 1.2
